### ISSUE TITLE
“牝”的其他读音：匕[bǐ]

### ISSUE THREAD

**Issue by ybay** [Dixin/Etymology#118]
If you have an issue with the website, please describe what is the issue, and how to reproduce the issue.

If you found an issue with etymology, please describe what is the Chinese character, and what is the issue or what is missing with the character or its etymology.

If you want a new feature, please describe how it works.

**Comment by ybay** [comment]
关于音的柱，“七聲”应该是“匕聲”。
(1)见网站内容“Shuowen 说文解字: 畜母也從牛七聲易曰畜牝牛吉”—>Shuowen 说文解字: 畜母也從牛匕聲易曰畜牝牛吉。
(2)见网站内容“匕”字部分：“Other pronunciations 其它发音: bǐ,pìn”。

**Comment by UncleHanzi** [comment]
Yes, correct 七 should be  匕 in the Shuowen 说文解字 file. I will have to get access to that file at some point. There are many typos in that file.

**Comment by UncleHanzi** [comment]
Replace 七 with 匕 in Shuowen table
